.travis.yml

```diff
@@ -2,10 +2,18 @@ language: node_js
 node_js:
   - "node"
   - "lts/*"
+services:
+  - docker
 cache: yarn
+before_install:
+  - docker run -d --rm -p 8554:8554 steabert/gst-rtsp-launch
```

Comment:
This container is left alive after the test is complete.
Is it possible to stop the container in after_script?

Reply:
I used the guidelines in the [docs](https://docs.travis-ci.com/user/docker/) and there they do not explicitely shut down the container. Is there any recommended way of cleaning up docker containers on Travis?